core/api/src/app/accounts/get-contacts-by-account.ts

```diff
@@ -0,0 +1,38 @@
+import { NoContactForUsernameError } from "@/domain/errors"
+
+import { ContactsRepository } from "@/services/mongoose"
+
+export const getContactByUsername = async ({
+  account,
+  contactUsername,
+}: {
+  account: Account
+  contactUsername: Username
+}): Promise<AccountContact | ApplicationError> => {
+  const contact = await ContactsRepository().findByHandle({
+    accountId: account.id,
+    handle: contactUsername,
+  })
+  if (contact instanceof Error) return new NoContactForUsernameError()
+
+  return {
+    id: contact.handle,
+    username: contact.handle,
+    alias: contact.displayName,
+    transactionsCount: contact.transactionsCount,
+  }
+}
+
+export const getContactsByAccountId = async (
+  accountId: AccountId,
+): Promise<AccountContact[] | ApplicationError> => {
+  const contacts = await ContactsRepository().listByAccountId(accountId)
+  if (contacts instanceof Error) return contacts
+
+  return contacts.map((contact) => ({
+    id: contact.handle,
+    username: contact.handle,
```

Comment:
same here

Reply:
that is what you need to validate with the e2e test, also it is not a problem if you dont create a breaking change in graphql